webapp/app/css/treeherder.css

```diff
@@ -972,6 +972,17 @@ ul.failure-summary-list li .btn-xs {
     margin-bottom: -1px;
 }
 
+.pinboard-related-bugs-btn > .pinboard-related-bugs-link,
+.pinboard-related-bugs-btn:hover > .pinboard-related-bugs-link:hover {
+    border-top-right-radius: 3px !important;
+    border-bottom-right-radius: 3px !important;
+}
+
+.pinboard-related-bugs-btn:hover > .pinboard-related-bugs-link {
+    border-top-right-radius: 0 !important;
+    border-bottom-right-radius: 0 !important;
+}
```

Comment:
I'm sure the css is required :) but wondering if you can elaborate on the need for it here and in the markup, and what it does for @camd when he gets to the review.

Wondering/asking if the `!important` is truly needed, just in case it can be omitted.